cfgov/unprocessed/js/config/error-messages-config.js

```diff
@@ -7,13 +7,18 @@
 'use strict';
 
 var ERROR_MESSAGES = {
-  CHECKBOX_ERRORS : {
-    required : 'Please select at least one of the "%s" options.'
+  CHECKBOX : {
+    required : 'Please select at least %s of the options.'
   },
-  DATE_ERRORS : {
+  DATE : {
     invalid : 'You have entered an invalid date.',
     one_required : 'Please enter at least one date.'
```

Comment:
Linter will complain about snake_case in JS. Also, should these be formatted as constants too (UPPERCASE) ?